R/prism_to_gdrDF.R

```diff
@@ -123,24 +127,29 @@ convert_LEVEL6_prism_to_gDR_input <- function(prism_data_path,
                      by = "row_name")
 
   full_data <- merge(full_data,
-                     treatment[, c("column_name", "broad_id", "name", "dose", "moa")],
+                     treatment[, intersect(names(treatment),
+                                           c("column_name", "broad_id", "name", "dose", "moa")),
+                               with = FALSE],
                      all.x = TRUE,
                      by = "column_name")
 
   full_data$value <- pmin(readout_min, 2 ^ full_data$value)
+  untrt_tag <- gDRutils::get_env_identifiers("untreated_tag")[1]
```

Comment:
👍 


(`"vehicle"` is characteristic for PRISM?)

Reply:
what do you mean characteristic? In gDR we use `vehicle` to indicate untreated controls

Reply:
```
> gDRutils::get_env_identifiers("untreated_tag")
[1] "vehicle"   "untreated"
```

And `"untreated"`?

Reply:
Yes, we use either "vehicle" or "untreated" to indicate untreated conditions